README.md

```diff
@@ -104,6 +110,42 @@ Using a custom hook prefix provides the ability to uniquely filter functionality
 
 The unique plugin slug is used by the telemetry server to identify the plugin regardless of the plugin's directory structure or slug.
 
+### Customize Opt-In Modal
+
+<img src="docs/modal.png" width="100%">
+
+The Opt-In modal contains plugin specific content including links for permissions, terms of service, and privacy policy which should be customized by using the [stellarwp/telemetry/{stellar_slug}/optin_args](#stellarwptelemetrystellar_slugoptin_args):
+```php
+add_filter( 'stellarwp/telemetry/my_stellar_slug/optin_args', 10, 1 );
+
+function my_custom_plugin_modal( $args ) {
+	// Customize all applicable arguments for the opt-in modal.
+	$args = [
+		'plugin_logo'           => Resources::get_asset_path() . 'resources/images/stellar-logo.svg',
+		'plugin_logo_width'     => 151,
+		'plugin_logo_height'    => 32,
+		'plugin_logo_alt'       => 'StellarWP Logo',
+		'plugin_name'           => 'The Events Calendar',
+		'plugin_slug'           => Config::get_container()->get( Core::PLUGIN_SLUG ),
+		'user_name'             => wp_get_current_user()->display_name,
+		'permissions_url'       => '#',
+		'tos_url'               => '#',
+		'privacy_url'           => '#',
+		'opted_in_plugins_text' => __( 'See which plugins you have opted in to tracking for', 'stellarwp-telemetry' ),
+		'heading'               => __( 'We hope you love {plugin_name}.', 'stellarwp-telemetry' ),
+		'intro'                 => __( 'Hi, {user_name}.! This is an invitation to help our StellarWP community. If you opt-in, some data about your usage of {plugin_name} and future StellarWP Products will be shared with our teams (so they can work their butts off to improve). We will also share some helpful info on WordPress, and our products from time to time. And if you skip this, that’s okay! Our products still work just fine.', 'stellarwp-telemetry' ),
+	];
+
+	return $args;
+}
+```
+
+### Post Integration Checklist
+
+- [ ] TBD
+- [ ] TBD
+- [ ] TBD
```

Comment:
Here are 3 I came up with... Not sure if they are good enough or not. 

1. Validate that the Optin Modal links are pointing to the correct URLs.
2. Confirm that you have set up marketing for the brand slug (ActiveCampaign or MailChimp currently).
3. Validate that the optin user is pushed to your marketing service.